app/poros/quote_search_result.rb

```diff
@@ -0,0 +1,10 @@
+class QuoteSearchResult
+  attr_reader :content,
```

Comment:
This looks good. Could you create a helper method in here that takes your tags out of an array and displays them as a comma separated string??